### DISCUSSION TITLE
Pop-up-effect for button

### DISCUSSION THREAD

**Discussion by ProPhycient5** [tailwindlabs/tailwindcss#10230] [Ideas]
Like there are many class names in the `Animation` section. So if you guys like you can add one more as `pop-up-effect` on click of a button.
```
.pop_up_effect {
  @apply transform transition duration-150 ease-out active:scale-90;
}
```

I have attached my code to👉🏽  this [link](https://play.tailwindcss.com/YErQLEEU57).
So please do and share your suggestion on this.

**Comment by Josianneshire** [comment]
I've tried running the binary file directly from the releases page as well.
You can check the output  [lcps go](https://www.lcpsgo.us/)
